src/platform/packages/shared/response-ops/rule_params/custom_threshold/v1.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { schema } from '@kbn/config-schema';
+import { COMPARATORS } from '@kbn/alerting-comparators';
+import { dataViewSpecSchema } from '../common';
+import { oneOfLiterals, validateKQLStringFilter, LEGACY_COMPARATORS } from '../common/utils';
+
+enum Aggregators {
+  COUNT = 'count',
+  AVERAGE = 'avg',
+  SUM = 'sum',
+  MIN = 'min',
+  MAX = 'max',
+  CARDINALITY = 'cardinality',
+  RATE = 'rate',
+  P95 = 'p95',
+  P99 = 'p99',
+  LAST_VALUE = 'last_value',
+}
+
+const comparators = Object.values({ ...COMPARATORS, ...LEGACY_COMPARATORS });
+
+const allowedAggregators = Object.values(Aggregators);
+allowedAggregators.splice(Object.values(Aggregators).indexOf(Aggregators.COUNT), 1);
```

Comment:
It is really strange for me to have this logic here. If we never use the element `COUNT` of `Aggregators` why not just do:

```
enum Aggregators { // or call it AllowedAggregators instead
  // COUNT = 'count',
  AVERAGE = 'avg',
  SUM = 'sum',
  MIN = 'min',
  MAX = 'max',
  CARDINALITY = 'cardinality',
  RATE = 'rate',
  P95 = 'p95',
  P99 = 'p99',
  LAST_VALUE = 'last_value',
}
```

Right now we are defining an `enum` that is not used anywhere else and then in the only place we need it we always remove an element. Feels wrong.

Reply:
Agree. Made changes according comments here: 06526098313a5842d1c8ff35d0eea664488d2030